Include/httpClient/httpClient.h

```diff
@@ -141,6 +144,21 @@ typedef enum HC_LOG_LEVEL
     LOG_ERROR,
 
     /// <summary>
+    /// Log warnings and errors
+    /// </summary>
+    LOG_WARNING,
```

Comment:
Why not typedef HCTraceLevel HC_LOG_LEVEL? So we don't have 2 types floating about that get used in different places

Reply:
I'd prefer no typedef and only one public type.  For a future PR

Reply:
HCTraceLevel is in the enum namespace in C, so we need to typdef it into a the type namespace anyway